packages/edge-login-ui-rn/src/components/common/MaterialInput.js

```diff
@@ -83,6 +84,7 @@ export class MaterialInput extends React.Component<Props, State> {
         : this.props.autoCorrect
     return (
       <TextField
+        testID={this.props.testID}
```

Comment:
Error. You will need to add `testID` to the file `flow-typed/react-native-material-textfield.js`